docs/oscar_references.bib

```diff
@@ -2230,6 +2230,21 @@ @Article{Rin13
   zbmath        = {6143064}
 }
 
+@article {Sau14,
+    AUTHOR = {Saunders, Neil},
+     TITLE = {Minimal faithful permutation degrees for irreducible {C}oxeter
+              groups and binary polyhedral groups},
+   JOURNAL = {J. Group Theory},
+  FJOURNAL = {Journal of Group Theory},
+    VOLUME = {17},
+      YEAR = {2014},
+    NUMBER = {5},
+     PAGES = {805--832},
+      ISSN = {1433-5883,1435-4446},
+       DOI = {10.1515/jgt-2014-0012},
+       URL = {https://doi.org/10.1515/jgt-2014-0012},
```

Comment:
```suggestion
```
bibtool complains that there is a doi in an URL field. It's enough to put it in the doi field, and DocumenterCitations.jl will take care of it.